src/main/java/io/cdap/plugin/gcp/bigquery/sink/AbstractBigQuerySinkConfig.java

```diff
@@ -84,6 +85,12 @@ public abstract class AbstractBigQuerySinkConfig extends BigQueryBaseConfig {
     "This value is ignored if the dataset or temporary bucket already exist.")
   protected String location;
 
+  @Name(NAME_JOB_LABEL_KEY_VALUE)
+  @Macro
+  @Nullable
+  @Description("Key value pairs to be added as labels to the BigQuery job.")
+  protected String jobLabelKeyValue;
```

Comment:
I don't see where `jobLabelKeyValue` is initialized.

Reply:
Fixed